dashboard/modules/usage_stats/usage_stats_head.py

```diff
@@ -0,0 +1,51 @@
+import asyncio
+import logging
+
+from concurrent.futures import ThreadPoolExecutor
+
+import ray
+
+import ray.dashboard.utils as dashboard_utils
+import ray._private.usage.usage_lib as ray_usage_lib
+
+from ray.dashboard.utils import async_loop_forever
+
+logger = logging.getLogger(__name__)
+
+
+class UsageStatsHead(dashboard_utils.DashboardHeadModule):
+    def __init__(self, dashboard_head):
+        super().__init__(dashboard_head)
+        self.cluster_metadata = None
+        self.session_dir = dashboard_head.session_dir
+
+    @async_loop_forever(ray_usage_lib._usage_stats_report_interval_s())
```

Comment:
You need to add a random offset here between [0, usage_stats_report_interval_s]. Otherwise, we'll have sampling bias and won't be able to accurately measure total session durations.

If you want a separate "session started" request, that should be a separate RPC and tagged as such.

Reply:
> If you want a separate "session started" request

This will be done in a follow up

> You need to add a random offset here between [0, usage_stats_report_interval_s]. Otherwise, we'll have sampling bias and won't be able to accurately measure total session durations.

Is this addressable if we just send a report upon termination? Or would there be other concerns still?

Reply:
Actually, can I do this in a follow-up PR? This requires some changes, and I think it is better splitting it. It also requires to change the schema from the server side, so I think it is better doing things in this way;

- merge this PR
- Update server side schema
- merge the next PR with seq no + cluster runtime

Reply:
Why's adding a random offset hard?

Reply:
It is not hard. I just thought it is needed for additional fields like cluster runtime (for the existing metadata, I don't think sample bias matters. Lmk if it is wrong), so I can add this to that PR since the current PR is already pretty big.